### ISSUE TITLE
Prioritize build matrix across projects

### ISSUE THREAD

**Issue by wr0ngway** [travis-ci/travis-ci#2038]
Frequently I test across multiple ruby versions, but only really care about one for my immediate gratification.  When working with multiple related repos, it takes a while for everything to run (especially for limited resources for private repos).  It would be great if I could give priority to specific subsets of the matrix, so for example if I pushed 5 repos, the mri 2.1.1 run would happen across all 5 repos before starting in on the lower priority jruby runs.  Ideally there would be an additional status of "Provisional success", so that I could see at a glance in the UI if my tests passed across all the repos for the high priority items.

**Comment by roidrage** [comment]
Closing this issue for now, as we have no immediate plans to add this feature.

Should we add it to the roadmap eventually, we'll make sure to update this ticket.
